lib/widgets/content.dart

```diff
@@ -866,7 +901,7 @@ class _BlockInlineContainerState extends State<_BlockInlineContainer> {
   @override
   Widget build(BuildContext context) {
     return InlineContent(recognizer: null, linkRecognizers: _recognizers,
-      style: widget.style, nodes: widget.nodes);
+      style: widget.style, nodes: widget.nodes, textAlign: widget.textAlign);
```

Comment:
Cool, I think this fixes https://github.com/zulip/zulip-flutter/pull/1031#discussion_r1855858138 .

Let's have a test case that would have caught that bug. Doesn't need to exercise all the possible alignments (that's covered by another test case you already have), just one of them.